packages/firestore/lib/modular/snapshot.d.ts

```diff
@@ -227,3 +227,30 @@ export declare function queryEqual<AppModelType, DbModelType extends DocumentDat
   left: Query<AppModelType, DbModelType>,
   right: Query<AppModelType, DbModelType>,
 ): boolean;
+
+/**
+ * Attaches a listener for a snapshots-in-sync event.
+ * The snapshots-in-sync event indicates that all listeners affected by a given change have fired, even if
+ * a single server-generated change affects multiple listeners.
+ *
+ * @param firestore
+ * @param observer
+ */
+export declare function onSnapshotsInSync(
```

Comment:
`onSnapshotsInSync` needs to be implemented in a separate PR. We don't have this functionality at the moment for some reason